src/network/networkprotocol.h

```diff
@@ -200,6 +200,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 		Unknown inventory serialization fields no longer throw an error
 		Mod-specific formspec version
 		Player FOV override API
+	UNVERSIONNED ADDITIONS (KIDSCODE):
```

Comment:
Unnecessary line. If your code does not need compatibility code depending on the network version, there's no need to increase it or state changes like this.